src/main/java/com/google/codeu/servlets/UfoDataServlet.java

```diff
@@ -0,0 +1,55 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Returns UFO data as a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
+ */
```

Comment:
Perhaps this comment/specification would be better suited to describe the doGet() method?

Reply:
Thanks! It is resolved.